src/e2e-test/java/io/cdap/plugin/utils/BigTableClient.java

```diff
@@ -0,0 +1,100 @@
+package io.cdap.plugin.utils;
+
+import com.google.api.gax.rpc.NotFoundException;
+import com.google.cloud.bigtable.admin.v2.BigtableInstanceAdminClient;
+import com.google.cloud.bigtable.admin.v2.models.CreateInstanceRequest;
+import com.google.cloud.bigtable.admin.v2.models.Instance;
+import com.google.cloud.bigtable.admin.v2.models.StorageType;
+import com.google.cloud.bigtable.hbase.BigtableConfiguration;
+import com.google.cloud.bigtable.hbase.BigtableOptionsFactory;
+import com.google.common.collect.ImmutableList;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.HColumnDescriptor;
+import org.apache.hadoop.hbase.HTableDescriptor;
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.Connection;
+import org.apache.hadoop.hbase.client.Put;
+import org.apache.hadoop.hbase.client.Table;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.codehaus.commons.nullanalysis.NotNull;
+
+import java.io.IOException;
+import java.util.List;
+import javax.annotation.Nullable;
+
+
+/**
+ * Represents GCP CBT Client.
+ */
+public class BigTableClient {
+    public static void createBigTableInstance(BigtableInstanceAdminClient adminClient,
+                                              String instanceId, String clusterId) {
+        if (!adminClient.exists(instanceId)) {
+            CreateInstanceRequest createInstanceRequest =
+                    CreateInstanceRequest.of(instanceId)
+                            .addCluster(clusterId, "us-central1-f",
+                                    3, StorageType.SSD)
+                            .setType(Instance.Type.PRODUCTION)
+                            .addLabel("it", "e2e-testing");
+            try {
+                Instance instance = adminClient.createInstance(createInstanceRequest);
+            } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
Please add a logger, and use `LOG.error("Error occuring while creating big table instance",e);`

Reply:
For reference : see class `GCSValidationHelper`